sd_jwt/examples/18013_5.yml

```diff
@@ -0,0 +1,45 @@
+user_claims:
+  iss: "https://example.com"
+  cnf:

```

Comment:
Is there reason to not let it use the holder_key from settings.yml? I think that's what will sign the KB-JWT anyway...